tests/benchmark_test_inputs/AB/mvs_config.json

```
@@ -0,0 +1,168 @@
+{
+    "economic_data": {
+        "currency": "EUR",
+        "discount_factor": {
+            "unit": "factor",
+            "value": 0.08
+        },
+        "label": "economic_data",
+        "project_duration": {
+            "unit": "year",
+            "value": 1
+        },
+        "tax": {
+            "unit": "factor",
+            "value": 0.0
+        }
+    },
+    "energyConsumption": {
+        "demand_01": {
+            "dsm": "False",
+            "energyVector": "Electricity",
+            "file_name": "demand.csv",
+            "inflow_direction": "Electricity",
+            "label": "Grid",
+            "type_asset": "demand",
+            "type_oemof": "sink",
+            "unit": "kW"
+        }
+    },
+    "energyConversion": {
+        "solar_inverter_01": {
+            "age_installed": {
+                "unit": "year",
+                "value": 0
+            },
+            "capex_fix": {
+                "unit": "currency",
+                "value": 0.0
+            },
+            "capex_var": {
+                "unit": "currency/kW",
+                "value": 3000
+            },
+            "efficiency": {
+                "unit": "factor",
+                "value": 1.0
+            },
+            "energyVector": "Electricity",
+            "inflow_direction": "PV plant (mono)",
+            "installedCap": {
+                "unit": "kW",
+                "value": 0.0
+            },
+            "label": "Solar inverter (mono)",
+            "lifetime": {
+                "unit": "year",
+                "value": 15
+            },
+            "opex_fix": {
+                "unit": "currency/kW/year",
+                "value": 0.0
+            },
+            "opex_var": {
+                "unit": "currency/kWh",
+                "value": 0.0
+            },
+            "optimizeCap": {
+                "unit": "bool",
+                "value": true
+            },
+            "outflow_direction": "Electricity",
+            "type_oemof": "transformer",
+            "unit": "kW"
+        }
+    },
+    "energyProduction": {
+        "pv_plant_01": {
+            "age_installed": {
+                "unit": "year",
+                "value": 0
+            },
+            "capex_fix": {
+                "unit": "currency",
+                "value": 10000
+            },
+            "capex_var": {
+                "unit": "currency/unit",
+                "value": 7200
+            },
+            "energyVector": "Electricity",
+            "file_name": "production.csv",
+            "installedCap": {
+                "unit": "kWp",
+                "value": 0
```

```suggestion
                "value": 5
```
To perform the benchmark test whose result can be calculated externally, you would need to define a fix installedCap.